test.py

```
@@ -9,11 +9,11 @@ def play_video(video_path):
         if os.name == 'nt':  # Windows
             player = r"C:\Program Files\VideoLAN\VLC\vlc.exe"
             if os.path.exists(player):
-                 subprocess.Popen([player, video_pathh])
+                 subprocess.Popen([player, video_path])
```

Ruff (0.12.2)

Undefined name `subprocess`

(F821)